pm_muairss_read/pm_muairss_read.xml

```diff
@@ -0,0 +1,44 @@
+<tool id="pm_uep_opt_read" name="Cluster outputs (pm-muairs_read)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        unzip $folder &&
+        out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
```

Comment:
Since we are now loading from a zip, we should validate that the unzipped folder does indeed contain input.cell and params.yaml (and add test cases where it doesn't).

I (accidentally) tried to run the tool with a folder that had neither of those, but it didn't trigger the FileNotFoundError:
```
  File "/home/qci88181/muon-galaxy-tools/pm_muairss_read/get_out_folder.py", line 25, in <module>
    sys.stdout.write(data['out_folder'])
NameError: name 'data' is not defined
```
But, thinking about it, it also shouldn't fall back on the default if it can't find the file at all.

Reply:
also added validation checks for input and param